build/publish/AzureInfo.targets

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+   
+  <Target Name="SetAzureProps"
+          Condition=" '$(IsOrchestratedPublish)' == 'True' And '$(PB_PublishBlobFeedUrl)' != '' ">
+
+    <ParseBlobUrl BlobUrl="$(PB_PublishBlobFeedUrl)">
+      <Output TaskParameter="BlobElements" ItemName="BlobElements" />
+    </ParseBlobUrl>
+
+    <PropertyGroup>
+      <PublishBlobFeedUrlBlobPath>%(BlobElements.BlobPath)</PublishBlobFeedUrlBlobPath>
+    </PropertyGroup>
+
+    <RegexReplace Input="$(PublishBlobFeedUrlBlobPath)" Expression="index.json" Replacement="" Count="1">
+      <Output TaskParameter="Output" ItemName="BlobPath" />
+    </RegexReplace>
+
+    <PropertyGroup>
+      <ArtifactContainerName>%(BlobElements.ContainerName)</ArtifactContainerName>
+      <ChecksumContainerName>%(BlobElements.ContainerName)</ChecksumContainerName>
+      <ArtifactCloudDropAccountName>%(BlobElements.AccountName)</ArtifactCloudDropAccountName>
+      <ChecksumCloudDropAccountName>%(BlobElements.AccountName)</ChecksumCloudDropAccountName>
+      <BlobStoragePartialRelativePath>%(BlobPath)assets</BlobStoragePartialRelativePath>
```

Comment:
We are sure the `/` is already in BlobPath?

Reply:
Yes, but Matt has a better way to remove the "index.json"